S/svg2pdf/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "svg2pdf"
+version = v"0.10.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/typst/svg2pdf.git", "04424512b519bc0b141775b2aee1e82c68293b5b")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/svg2pdf
+cargo build -p svg2pdf-cli --release
+install -Dvm 755 "target/${rust_target}/release/svg2pdf${exeext}" "${bindir}/svg2pdf${exeext}"
+install_license ${WORKSPACE}/srcdir/svg2pdf/LICENSE-MIT
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("x86_64", "macos"; ),
+    Platform("aarch64", "macos"; ),
+    Platform("x86_64", "freebsd"; ),
+    Platform("x86_64", "windows"; )
+]
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("svg2pdf", :svg2pdf)
```

Comment:
```suggestion
    ExecutableProduct("svg2pdf", :svg2pdf)
```
You have this listed as a library, but this appears to be an executable.

Reply:
Thanks a lot for this, I did not notice the issue, I fixed it in the latest commit